go.mod

```diff
@@ -41,6 +41,7 @@ require (
 	github.com/syndtr/goleveldb v1.0.1-0.20190318030020-c3a204f8e965
 	github.com/urfave/cli v1.22.9
 	github.com/whyrusleeping/timecache v0.0.0-20160911033111-cfcb2f1abfee
+	github.com/yusufpapurcu/wmi v1.2.2 // indirect
```

Comment:
if you remove this line and try `go mod tidy` again, will it disappear?

Reply:
nope, re-tried but no change